pkg/meta/base.go

```diff
@@ -951,3 +973,18 @@ func (m *baseMeta) doCleanupTrash(force bool) {
 		}
 	}
 }
+
+func (m *baseMeta) cleanupDelayedSlices() {
+	now := time.Now()
+	edge := now.Unix() - int64(m.fmt.TrashDays)*24*3600
+	logger.Debugf("Cleanup delayed slices: started with edge %d", edge)
+	if count, err := m.en.doCleanupDelayedSlices(edge, 3e5); err == nil {
```

Comment:
When there are more than 300K slices per hour, we can not cleanup them ASAP. Why do we need this limit?

Reply:
Same mechanism as cleaning up trash. But yes, we are lack of a way to delete the slices manually.